packages/api/src/controllers/auth.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Special controller for forwarding all incoming requests to a geolocated API region
+ */
+
+import { Router } from "express";
+import { authMiddleware } from "../middleware";
+import { db } from "../store";
+
+const app = Router();
+
+app.all(
+  "/",
+  authMiddleware({ originalUriHeader: "x-original-uri" }),
+  async (req, res) => {
+    const streamId = req.headers["x-livepeer-stream-id"];
+    const stream = await db.stream.get(streamId?.toString() ?? "");
```

Comment:
If `x-livepeer-stream-id` is absent this will throw error, and request will return 500 and some stack trace.
Wouldn't it be better to return 400 with clear message?